packages/next/src/client/components/globals/intercept-console-error.ts

```diff
@@ -28,11 +28,19 @@ export function patchConsoleError() {
 
     if (!isNextRouterError(maybeError)) {
       if (process.env.NODE_ENV !== 'production') {
+        // Create an origin stack that pointing to the origin location of the error
+        const originStack = (new Error().stack || '')
+          .split('\n')
+          // Remove error message and the stack of patched `window.console.error` call
+          .slice(2)
+          .join('\n')
```

Comment:
Let's give the console interception a name instead an use `Error.captureStackTrace` to cut of the frame. This is brittle under refactoring where somebody might move it into another function and then it breaks.